src/index.html

```diff
@@ -10,7 +10,49 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+
+    <form
+      action="/action_page.php"
+      method="post"
+      data-qa="big"
+      class="form"
+    >
+      <label>
+        <img
+          src="images/Search.svg"
+          alt="searchbar"
+          class="image"
+        >
+        <input
+          type="search"
+          class="search"
+          name="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </label>
+    </form>
+
+    <form
+      action="/action_page.php"
+      method="post"
+      data-qa="small"
+      class="form"
+    >
+      <label>
+        <img
+          src="images/Search.svg"
+          alt="searchsmallbar"
+          class="image smallimage"
```

Comment:
Apply ALL OVER the document
According to BEM documentation when you add modifier use like this 

```suggestion
          class="image image--small"
```